deployment/service.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
   name: mixer-service
   namespace: mixer
 spec:
-  type: NodePort
+  type: LoadBalancer
```

Comment:
For my understanding, what does this do?

Reply:
Here are the concept for NodePort and LoadBalancer:
https://kubernetes.io/docs/concepts/services-networking/service/#nodeport
https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer

They are types of kubernetes "services". NodePort does not have load balancer functions, and only expose a uniform port. LoadBlancer provisions a load balancer.

In GRPC + ESP scenario, we also have an ingress* sits in front of the Service (NodeType), that has load balancing, in the pure GRPC scenario, we do need the Service itself has load balancing.

* https://kubernetes.io/docs/concepts/services-networking/ingress/#what-is-ingress

Reply:
Thanks.